### ISSUE TITLE
Test build fails on iOS in two customer projects

### ISSUE THREAD

**Issue by jBorkowska** [leancodepl/patrol#1067]
For more info, contact @zoskar

**Comment by bartekpacia** [comment]
Some context: spent ~2 hrs trying to fix this bug on Monday (Match 6th), but all tricks known to me failed and the problem persists.

Gonna make another try soon.

**Comment by jBorkowska** [comment]
We encountered similar problem with @KmlBaranowski , pls contact him as well, maybe you'lol discover something there.

**Comment by bartekpacia** [comment]
```
    /Users/bartek/dev/leancode/REDACTED/mobile/ios/Runner/GeneratedPluginRegistrant.m:12:9: fatal error: module 'app_settings' not found
    @import app_settings;
     ~~~~~~~^~~~~~~~~~~~
    1 error generated.
```

**Comment by bartekpacia** [comment]
Second project (by @KmlBaranowski) fails with:

```
/Users/bartek/dev/leancode/REDACTED/mobile/ios/Runner/AppDelegate.swift:3:8: error: no such module 'GoogleMaps'
	import GoogleMaps
```

**Comment by bartekpacia** [comment]
oh man, I just wanted to give up and go to sleep, but then got the idea to pass `--configuration` it worked

@zoskar here's your magic line

```
$ patrol test -t integration_test/daily_mission_test.dart -v --configuration Debug-dev
```

**Comment by bartekpacia** [comment]
This was fixed in #1089 and released in [patrol_cli v1.1.0](https://pub.dev/packages/patrol_cli/changelog#110).
